src/setupTests.js

```diff
@@ -3,6 +3,16 @@ import 'blob-polyfill';
 
 import { toHaveNoViolations } from 'jest-axe';
 
+// VirtualizedSelect uses react-virtualized's AutoSizer to size the options menu.
+// Left to its own devices, in the unit test environment, AutoSizer makes the menu
+// list 0px wide and no options are rendered. Mocking AutoSizer makes the virtualized
+// window large enough for options to be rendered.
+jest.mock('src/components/common/VirtualizedSelectAutoSizer', () => {
```

Comment:
lets keep up the pattern of including the export type safety in this mock so that folks emulate the preferred pattern in their own mocks.  Yes, I know it's boilerplate, but the type safety is quite valuable to signal interface drift and avoid common unit testing headaches.

Reply:
This mock doesn't match the type. The previous mocks had `ts-expect-error` comments.

Reply:
I tried, but I wasn't able to figure out how to construct a mock that TS accepts.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4041/commits/470970b3c1b1306eaa58e1690d385f9ff83d063f after all.

Reply:
Wow